### PR TITLE
Fix regexp matching of multi-chunk inputs.

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

File: doc/autogen/types/sink.rst [hunk 1]
```diff
@@ -50,8 +50,8 @@
 
 .. spicy:method:: sink::gap sink gap False void (seq: uint<64>, len: uint<64>)
 
-    Reports a gap in the input stream. *seq* is the sequence number of the
```
Comment (rsmmr) [inline doc/autogen/types/sink.rst:53]: Have you checked what style is more common across all operators? I thought I had been meaning to use `*XX*` for  parameters, though obviously not consistently. Either way, if we make these changes, should ensure it's what we do elsewhere too.

File: hilti/runtime/src/tests/regexp.cc [hunk 2]
```diff
@@ -12,10 +13,17 @@
 using namespace hilti::rt;
 using namespace hilti::rt::bytes::literals;
 
+namespace hilti::rt::detail::adl {
+template<typename A, typename B>
+std::string to_string(const std::tuple<A, B>& xs, tag /*unused*/) {
```
Comment (rsmmr) [inline hilti/runtime/src/tests/regexp.cc:18]: Hmmm .. I'm surprised we need this, tuples should have a to_string() already. Also should output them as `(..)`.

File: doc/autogen/types/sink.rst (hunk 1 above)
Comment (rsmmr) [reply doc/autogen/types/sink.rst:53]: note: the changes below from single-tick to double-tickets are ok, they are just typos accidentally switching to markdown.

**Review by rsmmr — DISMISSED** [review]
Nice find. This looks good, except for the question on markup consistency.

File: doc/autogen/types/sink.rst (hunk 1 above)
Comment (bbannier) [reply doc/autogen/types/sink.rst:53]: I missed that, `*par*` seems indeed like the preferred way to refer to parameters. I rolled back the changes on that here.

**Pull request merged**